crud/common/utils.lua

```diff
@@ -199,11 +199,47 @@ function utils.tarantool_supports_uuids()
     return enabled_tarantool_features.uuids
 end
 
-function utils.convert_operations(user_operations, space_format)
-    if utils.tarantool_supports_fieldpaths() then
-        return user_operations
+local function id_not_in_table(operations, id)
+    for _, operation in ipairs(operations) do
+        if operation[2] == id then
+            return false
+        end
+    end
+
+    return true
+end
+
+local function add_nullable_fields_rec(operations, space_format, tuple, id)
+    if id < 2 or tuple[id - 1] ~= box.NULL then
+        return operations
+    end
+
+    if space_format[id - 1].is_nullable and id_not_in_table(operations, id - 1) then
+        table.insert(operations, {'=', id - 1, box.NULL})
+        return add_nullable_fields_rec(operations, space_format, tuple, id - 1)
+    end
+
+    return operations
+end
+
+function utils.add_intermediate_nullable_fields(operations, space_format, tuple)
+    if tuple ~= nil then
+        for i = 1, #operations do
+            operations = add_nullable_fields_rec(
+                operations,
+                space_format,
+                tuple,
+                operations[i][2]
+            )
+        end
+
+        table.sort(operations, function(v1, v2) return v1[2] < v2[2] end)
```

Comment:
You can't sort fields by number. Because it could be update by number or even jsonpath updates.

Reply:
Originally it was stated in issue - https://github.com/tarantool/crud/issues/113: "Field ''statistics'' was not found in the tuple"